### ISSUE TITLE
ChartJS: manual addition of data changes color for existing values

### ISSUE THREAD

**Issue by contactashish13** [Codeinwp/visualizer#517]
ref https://wordpress.org/support/topic/neat-but-a-tad-buggy/#post-11882227

**Comment by contactashish13** [comment]
PR changes the behavior of how defaults are applied to ChartJS charts. We will now **not** add defaults when charts are modified. We were adding default colors because ChartJS does not provide default colors when a new series is added and it appears grey. The problem with this is that if 
- a series is added
AND
- a config in advanced setting is modified

the latter overwrites the default color we had assigned to the series. This can be addressed by not giving defaults to existing series but that still does not solve the problem because, when we go to the advanced settings tab and change anything (say the chart title) the chart reloads with the new series in grey **not** the color we had assigned. Addressing this may complicate other things so it is safer to just ignore new series and let the user provide the colors.
